src/Management/src/TracingBase/TracingLogProcessor.cs

```diff
@@ -11,13 +11,11 @@ namespace Steeltoe.Management.Tracing
 {
     public class TracingLogProcessor : IDynamicMessageProcessor
     {
-        private readonly ITracing _tracing;
         private readonly ITracingOptions _options;
 
-        public TracingLogProcessor(ITracingOptions options, ITracing tracing)
+        public TracingLogProcessor(ITracingOptions options)
         {
             _options = options;
-            _tracing = tracing;
         }
 
         public string Process(string inputLogMessage)
```

Comment:
Does the below TODO still apply? that is, OTEL api still has this limitation?

Reply:
the context available here doesn't make it obvious if it is available... I have a recollection of reading that it still wasn't available, but I can look again

Reply:
ParentId is available on `Activity` but not anything Span related, so unless we switched over to reading the Activity.Current this TODO still applies